instrumentation/aws-java-sdk-sqs-1.10.44/build.gradle

```diff
@@ -8,7 +8,8 @@ dependencies {
 
 
 jar {
-    manifest { attributes 'Implementation-Title': 'com.newrelic.instrumentation.aws-java-sdk-sqs-1.10.44' }
+    manifest { attributes 'Implementation-Title': 'com.newrelic.instrumentation.aws-java-sdk-sqs-spans-1.10.44', 'Enabled': 'false',
```

Comment:
instead of disabling the module by default, why not add a config to enable adding the headers?

Reply:
This is the existing SQS instrumentation. Disabling would break customer dashboards

Reply:
Indeed, that is why I am recommending not disabling it by default, which your PR does.

Reply:
Also, I hadn't noticed, but you added spans to the module name.